sites/ubuntu.com.yaml

```diff
@@ -48,7 +48,7 @@ production:
     if ($host != 'ubuntu.com' ) {
       rewrite ^ https://ubuntu.com$request_uri? permanent;
     }
-    if ($request_uri = '/blog/latest-news') {
+    if ($uri = '/blog/latest-news') {
```

Comment:
Hey @nottrobin you have forgotten to do the changes also for staging

Reply:
Ah damn